src/nitpick/project.py

```diff
@@ -182,7 +181,7 @@ def merge_styles(self, offline: bool) -> Iterator[Fuss]:
 
         style = StyleManager(self, offline, config.cache)
         base = config.file.expanduser().resolve().as_uri() if config.file else None
-        style_errors = list(style.find_initial_styles(peekable(always_iterable(config.styles)), base))
+        style_errors = list(style.find_initial_styles(list(always_iterable(config.styles)), base))
```

Comment:
To be honest, I don't know why `peekable` was used in the first place.
It was not a change introduced by me, and I forgot to ask "why?" back then.
I assumed it was needed.